### ISSUE TITLE
User Information not updated on FindByLoginAsync

### ISSUE THREAD

**Issue by bagzli** [grandchamp/Identity.Dapper#72]
Steps to replicate:

Login user via external provider and then grab user information.

```
var info = await _signInManager.GetExternalLoginInfoAsync();
var result = await _signInManager.ExternalLoginSignInAsync(info.LoginProvider, info.ProviderKey, false);
var user = await _userManager.FindByLoginAsync(info.LoginProvider, info.ProviderKey);
```

The user object does not have its Id or any of the fields filled in. The login succeeds and the UserName field gets populated, but nothing else. Is this a bug or intended? If intended why is that so, is it possible to get it to fill out other information such as UserId?

**Comment by grandchamp** [comment]
Wanna create some integration tests to fix this behavior?

**Comment by grandchamp** [comment]
Fixed in #74
